App/Components/_appColors.js

```diff
@@ -0,0 +1,19 @@
+import colors from './_colors';
+
+const appColors = {
```

Comment:
There are several repeat values in appColors, is this for readability?  We could consolidate a few of them since this object could grow to be bigger than necessary when we continue to add components.